website/src/pages/showcase/index.tsx

```diff
@@ -81,136 +75,155 @@ function useSelectedTags() {
-    <div className="margin-top--l margin-bottom--md container">
-      <div className="row">
-        {TagList.map((tag) => {
-          const {label, description, icon} = Tags[tag];
+    <section className="container margin-top--xl margin-bottom--lg">
+      <div className={clsx('margin-bottom--sm', styles.filterCheckbox)}>
+        <span>
+          <h3>Filter</h3>
+          <p>{`(${filteredUsers.length} site${
+            filteredUsers.length > 1 ? 's' : ''
+          })`}</p>
+        </span>
+        <ShowcaseFilterToggle />
+      </div>
+      <ul className={styles.checkboxList}>
+        {TagList.map((tag, i) => {
+          const {label, description, color} = Tags[tag];
+          const id = `showcase_checkbox_id_${tag};`;
+
           return (
-            <div key={tag} className="col col--2">
-              <ShowcaseCheckbox
-                // TODO add a proper tooltip
-                title={`${label}: ${description}`}
-                name={tag}
-                label={
-                  icon ? (
-                    <>
-                      {icon} {label}
-                    </>
-                  ) : (
-                    label
-                  )
-                }
-                onChange={() => toggleTag(tag)}
-                checked={selectedTags.includes(tag)}
-              />
-            </div>
+            <li key={i} className={styles.checkboxListItem}>
+              <Tooltip id={id} text={description} anchorEl="#__docusaurus">
+                <ShowcaseTagSelect
+                  tag={tag}
+                  id={id}
+                  label={label}
+                  icon={
+                    tag === 'favorite' ? (
+                      <FavoriteIcon svgClass={styles.svgIconFavoriteXs} />
+                    ) : (
+                      <span
+                        style={{
+                          backgroundColor: color,
+                          width: 10,
+                          height: 10,
+                          borderRadius: '50%',
+                          marginLeft: 8,
+                        }}
+                      />
+                    )
+                  }
+                />
+              </Tooltip>
+            </li>
           );
         })}
-        <div className="col col--2">
-          <ShowcaseSelect
-            name="operator"
-            label="Filter: "
-            value={operator}
-            onChange={(e) => setOperator(e.target.value as Operator)}>
-            <option value="OR">OR</option>
-            <option value="AND">AND</option>
-          </ShowcaseSelect>
-        </div>
-      </div>
-    </div>
+      </ul>
+    </section>
   );
 }
 
-function ShowcaseCards({filteredUsers}: {filteredUsers: User[]}) {
+function ShowcaseCards() {
+  const selectedTags = useSelectedTags();
+  const filteredUsers = useFilteredUsers();
+  const favoriteUsers =
+    selectedTags.length === 0 &&
+    sortedUsers.filter((user) => user.tags.includes('favorite'));
```

Comment:
maybe sort once ahead of time?

when using filters, should favorites still be first in the list?

Reply:
> when using filters, should favorites still be first in the list?

I don't think so, favorites can be selected by selecting the favorite filter 

> maybe sort once ahead of time?

Yes, I'd update the component.